tests/rbd_mirror/test_rbd_group_mirror_misc.py

```diff
@@ -453,11 +481,208 @@ def test_rbd_group_mirror_unsupported_ops(
+            if err:
+                raise Exception("Failed to add group snapshot schedule of 1m")
+
+            status_spec_group = {
+                "pool": pool,
+                "group": pool_config.get("group"),
+                "format": "json",
+            }
+            if "namespace" in pool_config:
+                status_spec_group.update({"namespace": pool_config.get("namespace")})
+            else:
+                group_spec = pool + "/" + pool_config.get("group")
+
+            if verify_group_snapshot_ls(
+                rbd_primary, group_spec, "1m", **status_spec_group
+            ):
+                raise Exception("Failed to verify group snapshot schedule of 1m")
+            log.info(
+                "Verified Snapshot schedule of 1m set before client blocklisting is preserved"
+            )
+
+            snap_schedule_config.update({"level": "image", "image": images[0]})
+            out, err = add_snapshot_scheduling(rbd_primary, **snap_schedule_config)
+            if "is part of a group" in err:
+                log.info(
+                    "Image level snapshot scheduling failed as expected when group mirroring is enabled"
+                )
+            else:
+                raise Exception(
+                    "Image level snapshot schedule listed for the image when group mirroring is enabled"
+                )
+
+            status_spec = {"pool": pool, "format": "json", "image": images[0]}
+            if "namespace" in pool_config:
+                status_spec.update({"namespace": pool_config.get("namespace")})
+
+            out, err = rbd_primary.mirror.snapshot.schedule.ls(**status_spec)
+            if "part of a group" in err:
+                log.info(
+                    "Image level snapshot schedule failed as expected when group mirroring is enabled"
+                )
+            else:
+                raise Exception(
+                    "Image level snapshot schedule listed when group mirroring is enabled"
+                )
+
+            status_spec.pop("image")
+
+            if "namespace" in pool_config:
+                snap_schedule_config.update({"level": "namespace"})
+                out, err = add_snapshot_scheduling(rbd_primary, **snap_schedule_config)
+                if err:
+                    raise Exception(
+                        "Unable to add namespace level snapshot schedule of 1m when group mirroring is enabled"
+                    )
+                else:
+                    log.info(
+                        "Namespace level snapshot scheduling failed as expected when group mirroring is enabled"
+                    )
+
+                out, err = rbd_primary.mirror.snapshot.schedule.ls(**status_spec)
+                if err:
+                    raise Exception(
+                        "Namespace level snapshot schedule not listed when group mirroring is enabled"
+                    )
+                else:
+                    log.info(
```

Comment:
when it is not got set for namespace level, it should also fail to list in namespace level Is it?